testing/test_skipping.py

```diff
@@ -1084,7 +1084,7 @@ def test_func():
         result = testdir.runpytest("-rxs")
         result.stdout.fnmatch_lines(
             """
-            *XFAIL*
```

Comment:
Not sure about this change, all other verbose words are all caps, shouldn't we follow suit here?

Reply:
Yes, I was meaning to ask about that. I would like to change to `XFAIL` too, but this would be a change in the test report aswell (currently no change there, I'm only changing the short test summary). I'm unaware of the history about why `xfail` is currently lowercase in test report and all the others are uppercase.

On a related note, should the short test summary be colorized too?

Reply:
> On a somewhat related note, should the short test summary be colorized too?

Hah you read my mind, I was thinking just yesterday that colorizing the uppercase word (`XFAIL`, `FAIL`, etc) in the summary would make it easier to read. Please go ahead and let's see how it looks. 😄

Reply:
> When I get home I'll add a commit to change the xfail to XFAIL

Awesome, thanks!